code/create/dta_dicom2spec_rules.py

```diff
@@ -1,158 +1,115 @@
-## rules for converting the dta-dicoms into a bids-dataset.
-
+# rules for converting the dta-dicoms into a bids-dataset.
 from datalad_hirni.support.BIDS_helper import apply_bids_label_restrictions
 
 
 def _guess_modality(record, dcm_dict):
 
-    protocol = record.get("ProtocolName", None)
+    # not sure, if the strings in the dcm-headers are always consistent,
+    # so we just ignore the capital-letters.
+    protocol = record.get("ProtocolName", None).lower()
     if protocol:
 
-        # BEGIN Additional rule for forrest-structural
-        # TODO: Probably to be moved to some rule enhancement
-        if "VEN_BOLD" in protocol:
-            # TODO: Not clear yet; swi might be considered a datatype rather than
-            # a modality by respective BIDS extension:
-            # https://docs.google.com/document/d/1kyw9mGgacNqeMbp4xZet3RnDhcMmf4_BmRgKaOkO2Sc
-            return "swi"
-
-        if "DTI_" in protocol:
-            # TODO: What actually is the relevant part of protocol here?
+        if "dti_"   in protocol:
             return "dwi"
-
-        if "60dir".lower() in protocol.lower():
+        if "60dir"  in protocol:
             return "dwi"
-        if "120dir".lower() in protocol.lower():
+        if "120dir" in protocol:
             return "dwi"
-
         if "field map" in protocol:
             return "fieldmap"
-        # END
 
         import re
-        prot_parts = re.split('_|-|\s', protocol.lower())
+        prot_parts = re.split('_|-|\s', protocol)
 
         # this is a bad protocol name, but it is a resting state scan for pmu_rest and a task-bold if bold_task
-        if protocol.lower() in "dta_epi_pmu_rest" or protocol.lower() in "DTA_ep2d_bold_Task".lower():
+        if ("dta_epi_pmu_rest" == protocol) or ("DTA_ep2d_bold_Task".lower() == protocol):
                return "bold"
 
-########## Hier fuer DTA extra die Abfrage machen, ob Magnitude or phasediff
         image_type = record.get("ImageType", None)
-        if image_type and "fieldmap" in protocol.lower():
+        if image_type and "fieldmap" in protocol:
                 return "fieldmap"
 
-
-        # I deleted "t2star" from that list for this specific dataset, because it does not contain these.
-        # It would probably also be fine, if I had moved the "fieldmap" value to the start
-        direct_search_terms = ["t1", "t1w", "t2", "t2w",
-                               "t1rho", "t1map", "t2map", "flair",
-                               "flash", "pd", "pdmap", "pdt2", "inplanet1",
-                               "inplanet2", "angio", "dwi", "phasediff",
-                               "phase1", "phase2", "magnitude1", "magnitude2",
-                               "fieldmap", "epi", "meg"]
+	# this list should try to just contain values that might appear in the DTA-dataset.
+	# TODO: magnitude/phase does not appear in the dcm-headers, just fieldmap, and we correct it later?
```

Comment:
Yes, you cannot account for phase/magnitude herein. This is because we are generating a specification for a DICOM imageseries. That imageseries is neither phase or magnitude image - it's both.